src/main/resources/config/config.yaml

```diff
@@ -93,6 +93,10 @@ webhookMessageSendingQueueProcessorConfig:
   threadDelayInMilliseconds: ${WEBHOOK_MESSAGE_SENDING_QUEUE_THREAD_DELAY_IN_MILLISECONDS:-1000}
   httpClientConnectionPoolSize: ${WEBHOOK_MESSAGE_SENDER_CONNECTION_POOL_SIZE:-20}
   connectionPoolTimeToLive: ${WEBHOOK_MESSAGE_SENDER_CONNECTION_POOL_TIME_TO_LIVE:-60s}
+  connectionPoolIdleConnectionTimeToLive: ${WEBHOOK_MESSAGE_SENDER_CONNECTION_POOL_IDLE_CONNECTION_TIME_TO_LIVE:-5s}
   requestTimeout: ${WEBHOOK_MESSAGE_SENDER_REQUEST_TIMEOUT:-3s}
+  idleConnectionMonitorEnabled: ${WEBHOOK_MESSAGE_SENDER_IDLE_CONNECTION_MONITOR_ENABLED:-true}
```

Comment:
Do we want to enable it straightway in production as the default is `true`?

Reply:
good point, done.